Java/avg/array.java

```diff
@@ -12,4 +12,15 @@ public static void main(String[] args) {
             System.out.println(arr[i]);
         }
     }
+
+    public static void Even(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        int[] arr = new int[]{1, 2, 3, 4, 5};
+        System.out.println("Even numbers:");
+
+        for (int i = 0; i < arr.length; i = i * 2)
+        {
+            System.out.println(arr[i]);
+        }
+    }

```

Comment:
**Suggestion**: Add unit tests to verify the correct functionality of the Even (or renamed) method, especially focusing on edge cases such as empty arrays or arrays with a single element. <br> **Code Suggestion**: 
 ```
 +    @Test
 +    public void testPrintEvenNumbers() {
 +        // Implement unit tests
 +    }
 ```